code/modules/integrated_electronics/subtypes/input.dm

```diff
@@ -134,7 +136,9 @@
 	var/mob/living/carbon/human/H = get_pin_data_as_type(IC_INPUT, 1, /mob/living/carbon/human)
 	if(!istype(H)) //Invalid input
 		return
+
 	if(H in view(get_turf(H))) // Like medbot's analyzer it can be used in range..
```

Comment:
medibot too, but that might be for a seperate PR

Reply:
Oh, it really neccesary to fix such things in comments?

Reply:
Not really- just pointing it out.